Automattic/woocommerce-payments
# Add linting commands for JS, CSS, and PHP

Fixes #44

#### Changes proposed in this Pull Request

* Add linting commands to package.json

Setup is copied from [woocommerce/woocommerce-admin](https://github.com/woocommerce/woocommerce-admin).

Commands added:
- `npm run lint` for both JS and CSS
- `npm run lint:css`
- `npm run lint:css-fix` lint and fix CSS
- `npm run lint:js`
- `npm run lint:php`
- `npm run lint:php-fix` lint and fix PHP

#### Testing instructions

For all tests:

- `npm run lint` and check for errors
- `npm run lint:css` and check for _same_ CSS errors as `npm run lint`.
- `npm run lint:js` and check for _same_ JS errors as `npm run lint`.  

Tests:

* Have no linting issues and make sure linting commands don't show any issues.
* Have no CSS linting issues and make sure `npm run lint:css-fix` doesn't change any files.
* Have linting issues and make sure linting commands show issues.
* Have CSS linting issues and make sure `npm run lint:css-fix` fixes the issues.
